service/http/Readme.md

```diff
@@ -6,13 +6,35 @@ Start by importing Dapr go `service/http` package:
 daprd "github.com/dapr/go-sdk/service/http"
 ```
 
-## Event Handling 
+## Creating and Starting Service 
 
-To handle events from specific topic, first create a Dapr service, add topic event handler, and start the service:
+To create an HTTP Dapr service, first, create a Dapr callback instance with a specific address:
```

Comment:
nitpick: typo `an`

Reply:
Is that really a type? I'm not an English native speaker but I think the grammar rule says to use `an` before `a` word beginning with a vowel sound, not letter. Or did I get it wrong?